src/screens/MapViewScreen.js

```diff
@@ -53,14 +53,9 @@ export const MapViewScreen = ({ navigation, route }) => {
           >
             <WrapperRow spaceBetween>
               <View style={styles.augmentedRealityInfoContainer}>
-                {!!modelData.title && <RegularText big>{modelData.title}</RegularText>}
-                {!!modelData.locationInfo && (
-                  <RegularText small>{modelData.locationInfo}</RegularText>
-                )}
-                {!!modelData.description && (
-                  <View style={styles.marginTop}>
-                    <HtmlView html={modelData.description} tagsStyles={{ p: styles.p }} />
-                  </View>
+                {!!modelData?.title && <RegularText big>{modelData?.title}</RegularText>}
+                {!!modelData?.payload?.locationInfo && (
+                  <RegularText small>{modelData?.payload?.locationInfo}</RegularText>
```

Comment:
if you check for `!!modelData?.title` and `!!modelData?.payload?.locationInfo` why did you add the `?` in the rendering part `modelData?.title` and `modelData?.payload?.locationInfo`? what do you think is the benefit?

`modelData` is already checked at line 42, so it is safe to access `title` and `locationInfo` in these lines. the changes should be reverted in my opinion, or am i wrong?

removing the description rendering should have been the only change in this commit i suggest.

Reply:
Yes! You're right!!! 👍
But I'm still adding the undefined control of the payload

https://github.com/ikuseiGmbH/smart-village-app-app/pull/672/commits/07885128b69631df11a7404fd72a2c215a9f2e37